core/lib.rs

```diff
@@ -26,7 +26,7 @@ mod path;
 mod runtime;
 mod source_map;
 mod tasks;
-mod to_from_v8;
+pub mod to_from_v8;
```

Comment:
Make this public (and maybe rename it), or re-export the markers?

Reply:
I think we can make it public. Bikeshedding some names:

`v8_convert`, `convert`, `to_v8`

Reply:
Went with `convert`, I feel like `deno_core::convert::ToV8` reads nicely